qiskit_cold_atom/applications/fermi_hubbard.py

```diff
@@ -89,9 +89,7 @@ def __init__(
         self.particles_down = particles_down
         self.J = hop_strength
         self.U = int_strength
-        self.basis = FermionicBasis(
-            self.size, n_particles=[self.particles_up, self.particles_down]
-        )
+        self.basis = FermionicBasis(self.size, n_particles=[self.particles_up, self.particles_down])
```

Comment:
black used to force these linebreaks, so I'm wondering why this is now being changed? Has something changed in the black setup/preferences? (Is there even such a thing?)

Reply:
This depends on the line length. The checks that we have are taken from qiskit-experiments so we are consistent with other packages.